rasa/core/training/story_reader/markdown_story_reader.py

```diff
@@ -64,6 +61,10 @@ async def _process_lines(self, lines: List[Text]) -> List[StoryStep]:
                     continue
                 elif multiline_comment:
                     continue
+                elif line.startswith(">>"):
+                    # reached a new rule block
+                    rule_name = line[1:].strip("> ")
```

Comment:
Why not just simply `line.strip(">> ")`?

Reply:
@ricwo

Reply:
it's just as good - I copied what's done for story blocks (line 68)

Reply:
can we avoid mixing the two? so either use `line[3:]` or `line.strip(">> ")`?